includes/reports/reports-functions.php

```diff
@@ -1278,6 +1278,65 @@ function filter_items( $report = false ) {
 }
 add_action( 'edd_admin_filter_bar_reports', 'EDD\Reports\filter_items' );
 
+/**
+ * Get the order status array keys that can be used to run reporting related to gross reporting.
+ *
+ * @since 3.0
+ *
+ * @return array An array of order status array keys that can be related to gross reporting.
+ */
+function edd_gross_order_statuses() {
+	$statuses = array(
+		'completed',
+		'refunded',
+		'partially_refunded',
+		'revoked',
+	);
+
+	/**
+	 * Statuses that affect gross order statistics.
+	 *
+	 * This filter allows extensions and developers to alter the statuses that can affect the reporting of gross
+	 * sales statistics.
+	 *
+	 * @since 3.0
+	 *
+	 * @param array $statuses {
+	 *     An array of order status array keys.
+	 *
+	 */
+	return apply_filters( 'edd_gross_order_statuses', $statuses );
+}
+
+/**
+ * Get the order status array keys that can be used to run reporting related to net reporting.
+ *
+ * @since 3.0
+ *
+ * @return array An array of order status array keys that can be related to net reporting.
+ */
+function edd_net_order_statuses() {
+	$statuses = array(
+		'completed',
+		'partially_refunded',
+		'revoked',
```

Comment:
Should revoked really be included in the net statuses? I would have thought just `completed` and `partially_refunded`.

Reply:
> Revoked: access to purchased items is disabled, perhaps due to policy violation or fraud.

I don't know for sure, but it sounds like maybe this is a step you might use before refunded? You revoke while investigating, then later you might actually refund (or not). So at this point, while it's `revoked`, you haven't given the money back, so because you still have it, it would make sense to be in net?

Reply:
Thanks, that makes sense--I didn't realize the financial side of that status.

Reply:
Yep, the point of these are statuses as related to where the funds exist. You can use a Revoked if someone is violating your TOS, and is not entitled to a refund, so the money stays within your account as a store owner.